fury/pick.py

```diff
@@ -0,0 +1,66 @@
+import vtk
+
+
+class PickingManager(object):
+    def __init__(self, vertices=True, faces=True, actors=True,
+                 world_coords=True):
+        """ Picking Manager helps with picking 3D objects
+
+        Parameters
+        -----------
+        vertices : bool
+            If True allows to pick vertex indices.
+        faces : bool
+            If True allows to pick face indices.
+        actors : bool
+            If True allows to pick actor indices.
+        world_coords : bool
+            If True allows to pick xyz position in world coordinates.
+        """
+
+        self.pickers = {}
+        if vertices:
+            self.pickers['vertices'] = vtk.vtkPointPicker()
+        if faces:
+            self.pickers['faces'] = vtk.vtkCellPicker()
+        if actors:
+            self.pickers['actors'] = vtk.vtkPropPicker()
+        if world_coords:
+            self.pickers['world_coords'] = vtk.vtkWorldPointPicker()
+
+        # if self.mode == 'face':
+        #     self.picker = vtk.vtkCellPicker()
+        # elif mode == 'vertex':
+        #     self.picker = vtk.vtkPointPicker()
+        # elif self.mode == 'actor':
+        #     self.picker = vtk.vtkPropPicker()
+        # elif mode == 'world':
+        #     self.picker = vtk.vtkWorldPointPicker()
+        # else:
+        #     raise ValueError('Unknown picking option')
+
+    def pick(self, x, y, z, sc):
```

Comment:
- Can you add the docstring?
- to keep the same logic as some other function, I propose to do: 
` def pick(self, scene, point): ` -> point is a tuple or ndarray or list